excel-importer/src/main/java/uk/gov/hmcts/ccd/definition/store/excel/parser/AbstractDisplayGroupParser.java

```diff
@@ -152,39 +152,39 @@ public AbstractDisplayGroupParser(ParseContext parseContext, ShowConditionParser
-        this.userRoleColumn.ifPresent(column -> parseUserRole(caseType, group, groupDefinition.getValue()));
+        this.accessProfileColumn.ifPresent(column -> parseAccessProfile(caseType, group, groupDefinition.getValue()));
         return ParseResult.Entry.createNew(group);
     }
 
-    private void parseUserRole(CaseTypeEntity caseType,
+    private void parseAccessProfile(CaseTypeEntity caseType,
                                final DisplayGroupEntity group,
                                final List<DefinitionDataItem> groupDefinition) {
-        if (hasMultipleUserRoleRowsPerTab(groupDefinition)) {
+        if (hasMultipleAccessProfileRowsPerTab(groupDefinition)) {
             throw new MapperException(
-                String.format(
-                    "Please provide one user role row per tab in worksheet %s on column USER_ROLE for the tab %s",
-                    groupDefinition.get(0).getSheetName(), group.getReference()));
+                String.format("Please provide one access profile row per tab in worksheet %s on column ACCESS_PROFILE "
+                    + "for the tab %s", groupDefinition.get(0).getSheetName(), group.getReference()));
         }
         Optional<DefinitionDataItem> optionalDDI = groupDefinition.stream()
-            .filter(ddi -> StringUtils.isNotEmpty(ddi.getString(USER_ROLE))).findFirst();
+            .filter(ddi -> StringUtils.isNotEmpty(ddi.getString(ACCESS_PROFILE))).findFirst();
         if (optionalDDI.isPresent()) {
-            final String userRole = optionalDDI.get().getString(USER_ROLE);
-            group.setUserRole(getRoleEntity(caseType, group, groupDefinition, userRole));
+            final String accessProfile = optionalDDI.get().getString(ACCESS_PROFILE);
+            group.setAccessProfile(getAccessProfile(caseType, group, groupDefinition, accessProfile));
         }
     }
 
-    private UserRoleEntity getRoleEntity(CaseTypeEntity caseType,
-                                         DisplayGroupEntity group,
-                                         List<DefinitionDataItem> groupDefinition,
-                                         String userRole) {
-        return parseContext.getRole(caseType.getReference(), userRole)
+    private AccessProfileEntity getAccessProfile(CaseTypeEntity caseType,
+                                                 DisplayGroupEntity group,
+                                                 List<DefinitionDataItem> groupDefinition,
+                                                 String accessProfile) {
+        return parseContext.getAccessProfile(caseType.getReference(), accessProfile)
             .orElseThrow(() -> new MapperException(
-                String.format("- Invalid idam or case role '%s' in '%s' tab for TabId '%s'",
-                userRole, groupDefinition.get(0).getSheetName(), group.getReference())));
+                String.format("- Invalid IDAM access profile or case role '%s' in '%s' tab for TabId '%s'",
```

Comment:
Reword to remove IDAM

Reply:
Fixed in https://github.com/hmcts/ccd-definition-store-api/pull/961/commits/3951e2bdda580027372daf452a54fbcde899194f